client/verta/tests/operations/monitoring/test_summaries.py

```diff
@@ -42,16 +42,17 @@ def test_summary_labels(self, client):
         )
         assert isinstance(summary_sample, SummarySample)
 
+        with pytest.raises(TypeError):
+            float_histogram = data_types.FloatHistogram(
+                bucket_limits=[1, 13, 25, 37, 49, 61],
+                data=[15, 53, 91, 34, 7],
+            )
+            labels2 = {"env": "test", "color": "red"}
+            summary_sample_2 = summary.log_sample(
+                float_histogram, labels=labels2, time_window_start=yesterday, time_window_end=now
+            )
+            print("this should not be printed, and an error should be raised")
```

Comment:
Could we restrict the `pytest.raises` to just one of these statements (I think the `log_sample()`)?